### PR TITLE
Bug fixes 

### PR DESCRIPTION
**Pull request by neelam-kushwah**
Fixed few issues found while testing it with up-android-helloworld.
- Change payload format to UPAYLOAD_FORMAT_PROTOBUF_WRAPPED_IN_ANY
- Remove resource id and entity id from binder utransport as there is strict check to compare UURI in up-android-core, will revert once this fix is available in up-android-core
- Moved new setup.py to root directory
- lint issues

### CODE REVIEW

**Comment by neelam-kushwah** [comment]
@princemac, can you please review it.

**Review by princemac — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@ pure-python-adb==0.3.0.dev0
 Flask-SocketIO==5.3.6
 protobuf>=4.24.2
 pyaxmlparser==0.3.30
-up-python==0.1.2.dev0
\ No newline at end of file
+up-python==0.1.2.dev0
```
Comment (stevenhartley) [inline requirements.txt]: the semver standard we use would be more 0.1.2-dev, is the "." and the "0" required by pypi?

File: setup.py
Comment (stevenhartley) [inline setup.py:1]: Please do a git mv in lieu of copy & paste as we loose the history for the file.

File: simulator/tools/maxmin_field.py [hunk 2]
```diff
@@ -29,31 +29,31 @@
...
-    'running_duration': 16777215,
-    'temperature': 50,
-    'current_sequence': 7,
-    'total_sequences': 7,
-    'priority': 255,
-    'pressure': 20460,
-    'time_to_flat': 2097151,
-    'light_intensity': 1275,
-    'hours': 24,
+    "longitude": 180,
```
Comment (stevenhartley) [inline simulator/tools/maxmin_field.py:32]: Why these changes? Is there something with the single quote that is causing issues?

**Review by stevenhartley — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (neelam-kushwah) [reply requirements.txt]: Oh, I wasn't aware of this standard, the '0' isn't necessary, as it works with the 0.1.2-dev version too. I'll fix it

File: setup.py
Comment (neelam-kushwah) [reply setup.py:1]: will take care next time.

File: simulator/tools/maxmin_field.py (hunk 2 above)
Comment (neelam-kushwah) [reply simulator/tools/maxmin_field.py:32]: In Python, both single and double quotes work, but since we've configured Flake8 for linting, it suggests using double quotes.

**Review by stevenhartley — APPROVED** [review]
